docs/en/stack/ml/anomaly-detection/ml-revert-model-snapshot.asciidoc

```diff
@@ -0,0 +1,43 @@
+[role="xpack"]
+[[ml-reverting-model-snapshot]]
+= Reverting to a model snapshot
+
+{ml-docs}/ml-ad-run-jobs.html#ml-ad-model-snapshots[Snapshots of the {ml} model] 
+for each {anomaly-job} are saved frequently to an internal {es} index to ensure 
+resilience. It makes it possible to reset the model to a previous state in case 
+of a system failure or if the model changed significantly due to a one-off 
+event.
+
+. In {kib}, navigate to **{ml-app} > Anomaly Detection > Jobs**.
+. Locate the {anomaly-job} whose model you want to revert in the job table.
+. Open the job details and navigate to the **Model Snapshots** tab.
++
+--
+[role="screenshot"]
+image::images/anomaly-job-model-snapshots.jpg[A screenshot of a job with the Model Snapshots tab opened]
+--
+. Select the snapshot you want to revert from the list and click the **Revert** 
+  icon under **Actions**.
+. Optional: Select if you want to replay the analysis after the revert has been 
+  applied. You can decide whether you want the job running in real time. You can 
+  also define the time range you don't want the job to analyze, this way you can 
+  skip the time of the event you don't want to impact the model behavior. 
++
+--
+[role="screenshot"]
+image::images/revert-model-snapshot.jpg[A screenshot of a revert model snapshot flyout]
+--
+. Click **Apply**.
+
```

Comment:
I think it might also be a good idea to mention/link to custom rules in this page too. IIRC they are a way to tell the model to avoid updating the model in certain circumstances (and therefore potentially avoiding future situations where you'd need to revert).  There's a blurb about them in https://stack-docs_2156.docs-preview.app.elstc.co/guide/en/machine-learning/master/ml-ad-run-jobs.html#ml-ad-rules

Reply:
Added in [2381b34](https://github.com/elastic/stack-docs/pull/2156/commits/2381b347b6dbb6617ed11826718ad1e2296b74b9).